src/Middleware/src/Headstart.Common/Assets/english-translations.json

```diff
@@ -596,31 +587,27 @@
     },
     "CHECKOUT_SHIPPING": {
       "SAVE_CONTINUE": "Save and Continue",
-      "NO_RATES_RETURNED_MESSAGE": "No rates returned - shipping will be calculated after the order is placed",
-      "SHIP_ESTIMATES_BASED_ON_DELIVERY": "Shipping estimates are based on delivery speed after items have been shipped by the carrier. Items requiring customization could take longer to ship.",
-      "SHIPPING_SPEED": "Shipping Speed"
+      "SHIPPING_SPEED": "Make Shipping Speed Selections To Continue"
     },
     "SHIPPING_SELECTION_FORM": {
       "DAY": "day",
       "DELIVERY_OPTIONS": "Delivery Options",
       "HOW_FAST": "How fast should this ship?",
-      "SHIPPING_SPEED": "Shipping Speed"
+      "SHIPPING_SPEED": "Shipping Speed",
+      "NO_RATES_RETURNED_MESSAGE": "No rates returned - shipping will be calculated after the order is placed",
+      "SHIP_ESTIMATES_BASED_ON_DELIVERY": "Shipping estimates are based on delivery speed after items have been shipped by the carrier. Items requiring customization could take longer to ship."
     }
   },
   "COMMON": {
-    "ADD": "Add",
```

Comment:
can we validate we want these common translations removed? It seems like we would need Add Approve Decline and Save

Reply:
Validated - I'm not seeing any of these referenced anywhere in the templates. They were likely moved under something more specific (not COMMON)